components/style/keyframes.rs

```diff
@@ -82,6 +84,18 @@ pub struct Keyframe {
     pub block: Arc<RwLock<PropertyDeclarationBlock>>,
 }
 
+impl ToCss for Keyframe {
```

Comment:
This doesn't write out the property declaration block.

https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes

A single keyframe rule looks like `68%, 72% { left: 50px; top: 50px; }`. So you first print out a comma-separated list of percentages, and then serialize the block. I think you'll need to add the braces yourself.